stock_packaging_usability/models/stock_pack_operation.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# © 2014-2018 Akretion (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models, _
+from odoo.exceptions import UserError
+from odoo.tools import float_compare, float_is_zero
+
+
+class StockPackOperation(models.Model):
+    _inherit = 'stock.pack.operation'
+
+    def put_in_new_pack(self):
+        self.ensure_one()
+        pack = self.env['stock.quant.package'].create({})
+        self.put_in_target_pack(pack.id)
+
+    def put_in_last_pack(self):
+        all_cur_packs_ids = [
+            packop2.result_package_id.id for packop2
+            in self.picking_id.pack_operation_ids if packop2.result_package_id]
+        if not all_cur_packs_ids:
+            raise UserError(_('There is no current package.'))
+        pack_id = max(all_cur_packs_ids)
+        self.put_in_target_pack(pack_id)
+
+    def put_in_target_pack(self, pack_id):
+        self.ensure_one()
+        assert pack_id, 'pack_id is a required argument'
```

Comment:
@alexis-via This won't never be launched as pack_id is a required argument

Reply:
yes, it can be launched if you call the with `pack_id` being anything that evals to `False` like None, 0 , etc.